samples/bluetooth/channel_sounding_ras_initiator/src/distance_estimation.c

```diff
@@ -21,7 +21,8 @@ LOG_MODULE_DECLARE(app_main, LOG_LEVEL_INF);
 #define SPEED_OF_LIGHT_M_PER_S	(299792458.0f)
 #define SPEED_OF_LIGHT_NM_PER_S (SPEED_OF_LIGHT_M_PER_S / 1000000000.0f)
 #define PI			3.14159265358979323846f
-#define MAX_NUM_SAMPLES		256
+#define MAX_NUM_RTT_SAMPLES		256
+#define MAX_NUM_IQ_SAMPLES		256 * CONFIG_BT_RAS_MAX_ANTENNA_PATHS
```

Comment:
it's not really related to this PR but I suppose another improvement would be to process the tone extension slots